import sys

raise ValueError('bad')
sys.stdout.write('end')
